README.md

```diff
@@ -0,0 +1,31 @@
+# Mapeo Map Server
+
+An in-progress offline map style and tile server for Mapeo
+
+## Technical Details
+
+- [Fastify](https://fastify.io/) for creating the server.
+- [SQLite](https://sqlite.org/index.html) as the underlying persistence layer.
+- [Prisma](https://www.prisma.io/) as a **build** tool for updating the schema, creating migration scripts, and generating schema assets.
+  - Schema diagram can be found [here](/prisma/ERD.svg).
+  - Due to the distributed and local-first nature of Mapeo, migrations are performed at runtime by the server on initialization. See the [migrations implementation](/src/lib/migrations.ts) for more details. It roughly follows the logic that Prisma uses for build-time migrations.
+
+## Features
```

Comment:
left these as unchecked for now as i just wanted to list out what the goals of this project were. can check them off as appropriate in a follow-up PR when its closer to being ready for use